tests/bearlib/aspects/conftest.py

```diff
@@ -76,3 +78,13 @@ class SubAspect:
         sour = SubAspect_tastes['sour']
 
     return SubAspect
+
+
+@pytest.fixture
+def aspectsTestBear():
+    class testBear(LocalBear, aspects={
+        'detect': [Metadata.CommitMessage.Shortlog],
+        'fix': [Metadata.CommitMessage.Shortlog.TrailingPeriod]},
+                      languages=['Python']):
```

Comment:
@Techievena You make me very proud ;) but we should all provide our opinions on that :)

@coala/aspects-developers